<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widgetUserInfo" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>480</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelHead">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelLastTime">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>746</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetBook" native="true">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>0</y>
      <width>601</width>
      <height>480</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelDateTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>日期：</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidgetPerson">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="labelBookPerson">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>分账人：</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPS">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBook">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>入账</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCanel">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMoney">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>金额：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMoney">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>145</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>0</y>
       <width>3</width>
       <height>450</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>610</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPic">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>430</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>账单图片</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditPs">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <zorder>lineEditMoney</zorder>
    <zorder>labelDateTime</zorder>
    <zorder>treeWidgetPerson</zorder>
    <zorder>labelBookPerson</zorder>
    <zorder>labelPS</zorder>
    <zorder>pushButtonBook</zorder>
    <zorder>pushButtonCanel</zorder>
    <zorder>labelMoney</zorder>
    <zorder>dateTimeEdit</zorder>
    <zorder>line_3</zorder>
    <zorder>line_7</zorder>
    <zorder>pushButtonPic</zorder>
    <zorder>textEditPs</zorder>
   </widget>
   <widget class="QWidget" name="widgetHistory" native="true">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>0</y>
      <width>601</width>
      <height>480</height>
     </rect>
    </property>
    <widget class="QPushButton" name="pushButtonFirst">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>430</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPre">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>430</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNext">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>430</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLast">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>430</y>
       <width>15</width>
       <height>15</height>
      </rect>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeWidgetHistory">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>11</y>
       <width>571</width>
       <height>401</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>0</number>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="labelPage">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>页</string>
     </property>
    </widget>
    <widget class="QLabel" name="Textlabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>430</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>0</y>
       <width>3</width>
       <height>450</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>610</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelLoading">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widgetOut" native="true">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>0</y>
      <width>601</width>
      <height>480</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>0</y>
       <width>3</width>
       <height>450</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>610</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowserOut">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>471</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actionKeepin">
   <property name="text">
    <string>账单</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>设置</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>bookKeeping</string>
   </property>
  </action>
  <action name="actionOutBookKeeping">
   <property name="text">
    <string>outBookKeeping</string>
   </property>
  </action>
  <action name="actionBookKeepingHistory">
   <property name="text">
    <string>bookKeepingHistory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
